hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestHSync.java

```diff
@@ -227,7 +235,7 @@ public static void init() throws Exception {
         (OpenKeyCleanupService) cluster.getOzoneManager().getKeyManager().getOpenKeyCleanupService();
     openKeyCleanupService.suspend();
 
-    preFinalizationChecks();
+//    preFinalizationChecks();
```

Comment:
[HDDS-11292](https://issues.apache.org/jira/browse/HDDS-11292) fixed this problem. Need to rebase for this.